ratatui/src/lib.rs

```diff
@@ -79,7 +77,27 @@
 //! You can also watch the [FOSDEM 2024 talk] about Ratatui which gives a brief introduction to
 //! terminal user interfaces and showcases the features of Ratatui, along with a hello world demo.
 //!
-//! ## Introduction
+//! # Crate Organization
+//!
+//! Starting with Ratatui 0.30.0, the project was reorganized into a modular workspace to improve
+//! compilation times, API stability, and dependency management. Most applications should continue
+//! using this main `ratatui` crate, which re-exports everything for convenience:
+//!
+//! - **[`ratatui`](crate)**: Main crate with complete functionality (recommended for apps)
+//! - **[`ratatui-core`]**: Core traits and types for widget libraries
+//! - **[`ratatui-widgets`]**: Built-in widget implementations
+//! - **Backend crates**: [`ratatui-crossterm`], [`ratatui-termion`], [`ratatui-termwiz`]
+//! - **[`ratatui-macros`]**: Procedural macros and utilities
```

Comment:
There are no _procedural_ macros (at least not yet).
```suggestion
//! - **[`ratatui-macros`]**: Macros for simplifying the boilerplate
```